.github/workflows/actions.yml

```diff
@@ -230,14 +230,15 @@ jobs:
         sudo apt update
         sudo apt install -y ninja-build gcc-multilib g++-multilib
         sudo pip install py-markdown-table
+        sudo pip install tqdm
```

Comment:
Why do we need this? Could we avoid it easily?

Reply:
It's needed for printing the progress bar in _non-verbose_ mode. With it, it's much easier to print nice progress bar. Like `sudo pip install py-markdown-table` (previous line), it's a _comfort option_. Not necessary but useful.

Is it fine or I should remove it?

Reply:
Let us wait for others opinion.

Reply:
It's fine!